@xen-orchestra/web-core/lib/components/tag/VtsTagList.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div class="vts-tag-list">
+    <UiTag v-for="tag in tags" :key="tag" :accent :variant size="small">{{ tag }}</UiTag>
```

Comment:
I think a `<slot>` is more appropriate here, since we may need to use multiple tags with different `accent` and / or `variant` in the same list instance.
```suggestion
    <slot />
```